ui/src/app/edge/live/storage/modal/modal.component.ts

```diff
@@ -31,13 +31,13 @@ export class StorageModalComponent implements OnInit {
         this.service.getConfig().then(config => {
             this.config = config;
             let channels = [];
-            this.chargerComponents = config.getComponentsImplementingNature("io.openems.edge.ess.dccharger.api.EssDcCharger");
+            this.chargerComponents = config.getComponentsImplementingNature("io.openems.edge.ess.dccharger.api.EssDcCharger").filter(component => !component.isEnabled == false);
```

Comment:
this.chargerComponents = config.getComponentsImplementingNature("io.openems.edge.ess.dccharger.api.EssDcCharger").filter(component => component.isEnabled);